internal/orchestrator/updates.go

```diff
@@ -567,10 +567,24 @@ func (o *Orchestrator) eventUpdate(ctx context.Context, evt *v1types.ConditionUp
 }
 
 func (o *Orchestrator) finalizeCondition(ctx context.Context, cond *rctypes.Condition) error {
+	// if we fail to update the event history or to delete this event from the KV,
+	// the reconciler  will catch it later and walk this code, so return early. It
+	// is kosher to replay event history iff the contents of that history (id,
```

Comment:
`iff` should be `if`?

Reply:
I was lazy and geeky and mean it in the "if and only if" sense. Finalizing a condition is idempotent if the non-time contents of the event are identical. If they aren't it's an error.

Reply:
Ah, ok. I haven't seen that notation before.